developments_build/sql/aggregate.sql

```diff
@@ -0,0 +1,309 @@
+DROP TABLE IF EXISTS _AGGREGATE_block;
+WITH
+bctcb2010_aggregate AS (
+    SELECT 
+        boro,
+        bctcb2010,
+        cenblock10,
+        bct2010,
+        centract10,
+        nta2010,
+        ntaname2010,
+        --puma2010,
+        --comunitydist,
+        --councildist,
+        SUM(comp2010ap) as comp2010ap,
+        SUM(comp2010) as comp2010,
+        SUM(comp2011) as comp2011,
+        SUM(comp2012) as comp2012,
+        SUM(comp2013) as comp2013,
+        SUM(comp2014) as comp2014,
+        SUM(comp2015) as comp2015,
+        SUM(comp2016) as comp2016,
+        SUM(comp2017) as comp2017,
+        SUM(comp2018) as comp2018,
+        SUM(comp2019) as comp2019,
+        SUM(comp2020q2) as comp2020q2,
```

Comment:
`comp2020q2` feels weird to me, maybe we should raise to HED, this field should just be called `comp2020` instead?

Reply:
I think it makes sense. We don't have full data for 2020.

Reply:
The only thing that is potentially confusing is that it is both q1 & q2. It should be `comp2020_to_date` or something.